.github/workflows/dockerhub-master.yml

```diff
@@ -12,19 +12,24 @@ jobs:
     steps:
       - uses: actions/checkout@v2
 
-      - name: build-tag-push
-        uses: docker/build-push-action@v1
-        id: docker
+      - name: Login to DockerHub
+        uses: docker/login-action@v1
         with:
           username: ${{ secrets.DOCKER_USERNAME }}
           password: ${{ secrets.DOCKER_PASSWORD }}
-          dockerfile: docker/Dockerfile
-          cache_froms: buildpack-deps:stable-curl
-          repository: xaynetwork/xaynet
-          tag_with_ref: true
-          tags: development
+
+      - name: Set up Docker Buildx
+        uses: docker/setup-buildx-action@v1
+
+      - name: build-tag-push
+        uses: docker/build-push-action@v2
+        id: docker
+        with:
+          context: .
+          file: docker/Dockerfile
+          tags: xaynetwork/xaynet:development
           push: true
-          build_args: COORDINATOR_FEATURES=metrics
+          build-args: COORDINATOR_FEATURES=metrics
 
       - name: Install doctl
```

Comment:
This will soon change as we'll move away from DO in favor of IONOS, but we change this when we get to that, no worries. Just mentioning.